aries-site/src/layouts/content/Subsection.js

```diff
@@ -78,7 +78,7 @@ export const Subsection = ({
           <Header>
             <Box align="start" gap="small">
               {level === 1 && topic && (
-                <Link href={`/${topic.toLowerCase()}`} passHref>
+                <Link href="/[topic]" as={`/${topic.toLowerCase()}`} passHref>
```

Comment:
Not sure I understand how this change works, can you elaborate?

Reply:
The href tells text JS that we are going to use a dynamic route where the template is [topic].js, but we supply `as` so the browser knows what the value of the path will be. Described here: https://nextjs.org/docs/api-reference/next/link#dynamic-routes